src/app/globals.css

```diff
@@ -3,3 +3,19 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+a {
+  text-decoration: none;
+}
+html,
+h1 {
+  margin: 0;
+}
```

Comment:
zastanawiam się czy finalnie jest to potrzebne, tailwind nie ma przypadkiem wbudowanych tzw. "resetów"?

Wiem że ja Ci to sugerowałem ale po tym jak znaleźliśmy gdzie był błąd z itemem to można to chyba usunąć